lib/util.js

```diff
@@ -37,6 +38,9 @@ util.cleanAnonymous = async () => {
         const arangoDbExists = await arango.checkIfDatabaseExists(username);
         if (arangoDbExists) await arango.deleteAccount(username);
 
+        const igniteDbExists = await ignite.checkAccount(username);
+        if(igniteDbExists) await ignite.deleteAccount(username);
```

Comment:
Can you add a space after the `if`? For codebase styling consistency. We have a `prettierrc.js` file that is supposed to take care of styling. Are you using prettier?

https://github.com/garageScript/databases/blob/master/.prettierrc.js

Reply:
I use prettier as default vscode formatter usually, so I forgot to check these files before submitting. I run prettier over all database files in new commit and it picked up a few inconsistencies here and there, mostly empty spaces around objects. 
Everything should be fixed now.